### PR TITLE
Blog app - Controllers

### PR DESCRIPTION
**Pull request by itsmraga-hub**
#  💯 Achievements

- Use controllers to handle requests and render empty views.
- Understand Rails naming conventions.
- Use params from browser requests in a safe way.

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,11 @@
 Rails.application.routes.draw do
+  # get 'users/index'
+  get 'users/', to: "users#index", as: 'users'
+  get 'users/:id', to: "users#show", as: 'user'
+  get 'users/:user_id/posts', to: "posts#index", as: 'user_posts'
+  get 'users/:user_id/posts/:id', to: "posts#show", as: 'user_post'
```
Comment (gemmen29) [inline config/routes.rb]: - [x] [OPTIONAL] Kindly use the following way to define your routes as it's more efficient and easier to read.
 ```ruby
  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end

```

**Review by gemmen29 — APPROVED** [review]
Hi @itsmraga-hub, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 



## Optional suggestions:
- [ ] __Check the comments under the review.__

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: config/routes.rb (hunk 1 above)
Comment (itsmraga-hub) [reply config/routes.rb:6]: Alright, thanks. I thought that was for the next sage but will do it.

Comment (itsmraga-hub) [reply config/routes.rb]: Alright, thanks. I thought that was for the next stage but will do it.

**Pull request merged**